bears/yml/RAMLLintBear.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='ramllint',
+        use_stdout=False,
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(?P<severity>error) (?P<filename>.+)\n  (?P<message>.+)')
```

Comment:
yes, you're right, it takes 3, initially I had them but because it didnt catch any regex I tried with only one as i knew for sure that the file im testing has only errors :D thanks